pyshelf/cloud/factory.py

```diff
@@ -13,10 +12,6 @@ def create_storage(self, bucket_name):
         # required and is not used because we want the ability to change
         # buckets when we want.
         bc = utils.get_bucket_config(self.config, bucket_name)
-        if bc is None:
```

Comment:
Lol I see what you were saying now. I move this shit around so it made more sense let me know what you think? b9a6a8e7cb83a6dab7bd6a04da5466c5897717ef